src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -151,6 +151,16 @@ com/sun/ts/tests/jaspic/spi/baseline = jaspic jaspic_baseline
 com/sun/ts/tests/javamail = javamail javaee javamail_web_profile javaee_web_profile_optional
 com/sun/ts/tests/javaee = javaee
 com/sun/ts/tests/jaxrs = jaxrs javaee jaxrs_web_profile javaee_web_profile
+com/sun/ts/tests/jaxrs/jaxrs21/ee/sse/sseeventsource = jaxrs javaee jaxrs_web_profile javaee_web_profile xml_binding
```

Comment:
If these tests are using jakarta.xml.bind package then shouldn't it be javaee_optional?

Reply:
yes, I will address this.

Reply:
Addressed with new commit.